pkg/expapi/types.go

```diff
@@ -309,8 +309,8 @@ type DaemonSpec struct {
 	Template *api.PodTemplateSpec `json:"template,omitempty"`
 }
 
-// DaemonStatus represents the current status of a daemon.
-type DaemonStatus struct {
+// DaemonSetStatus represents the current status of a daemon.
+type DaemonSetStatus struct {
 	// CurrentNumberScheduled is the number of nodes that are running exactly 1 copy of the
 	// daemon and are supposed to run the daemon.
```

Comment:
this sentence is exactly why it's good we're changing daemon to daemon set. this is a case where "daemon" really means "daemon" and not "daemon set" :)